Source/Fuse.Scripting.JavaScript/Tests/UX/ClearBindingTest.ux

```diff
@@ -8,19 +8,19 @@
 			p.removeAt(0);
 		}
 
-		module.exports = { p, remove }
+		module.exports = { p: p, remove: remove };
```

Comment:
Adding the semicolon shouldn't be necessary

Reply:
Not necessary but less chance of getting bitten by js' auto-semicolon rules when refactored in future. Seemed like a painless improvement given that I was having to review all the code anyway

Reply:
Please don't. This is not the right PR to discuss JS code-style.

Reply:
No probs, I'll remove them :)